_posts/2019-09-04-pulp-3-performance-blog.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Pulp 3 Performance Blog
+author: Imaanpreet Kaur
+team: Red Hat Performance Team
+members: Pradeep Surisetty, Jan Hutař, Imaanpreet Kaur
+---
+
+## Motivation
+
+As we all know, Pulp3 is under active development. So, it's predominant  to check the performance of the pulp3. Pulp team come up with a test plan and Red Hat Performance team tested these test cases. So, let’s discuss about the pulp3 performance in brief.
```

Comment:
Can we replace `As we all know, Pulp3 is under active development. So, it's predominant  to check the performance of the pulp3.` with:

```
We want to investigate the performance of various Pulp3 operations including: Sync, Publish, Download, Content Listing, and Copy operations.
```

Reply:
Also `Pulp team come up` should be `The Pulp team came up` to be the past tense and use the Definite article, `The`.